src/apps/vpn/platforms/android/androidvpnactivity.h

```diff
@@ -48,6 +48,11 @@ enum ServiceAction {
   ACTION_GLEAN_SET_SOURCE_TAGS = 14,
   // Set startOnBoot pref
   ACTION_SET_START_ON_BOOT = 15,
+  // Reactivate the last connection (unusued from the client)
+  ACTION_REACTIVATE = 16,
```

Comment:
Why do we need this if it is unused?

Reply:
Completeness. it's part of our ipc protocol but not used in c++ (bot in kotlin-ipc).
I don't want to look in which file the magic integer is defined :D